Documentation/release-notes/3964.md

```diff
@@ -0,0 +1,21 @@
+### Add new AOT profiling targets
+
+The new `BuildAndStartAotProfiling` target builds the package with
+embedded AOT profiler, sets AOT profiler socket port to
+`$(AndroidAotProfilerPort)` and starts the *launch* activity.
+
+The new `FinishAotProfiling` target collects the AOT profiler data
+from the device or the emulator through sockets port
+`$(AndroidAotProfilerPort)` and writes them to
+`$(AndroidAotCustomProfilePath)`.
+
+The default values for port and custom profile are `9999` and
+`custom.aprof`.
+
+The `aprofutil` call may be extended with
+`$(AProfUtilExtraOptions)`, to pass additional options.
```

Comment:
should we be consistent about our property naming. We have `AndroidAotProfilerPort` and `AndroidAotCustomProfilePath` which match all of our other (new) properties. But we have `AProfUtilExtraOptions`.
Should it be `AndroidAProfUtilExtraOptions` or `AndroidProfUtilExtraOptions` ?

Reply:
`AndroidAProfUtilExtraOptions` seemed too long and as it was already suggested and thumbed up by @jonpryor without the Android prefix, I went ahead with that :-)

Note that the `A` in `AProf` is for AOT.